app/callbacks/__init__.py

```diff
@@ -0,0 +1,13 @@
+from functools import partial
+
+from flask import Blueprint
+
+from app.authentication import requires_authentication
+
+callbacks = Blueprint('callbacks', __name__, url_prefix='/callbacks')
+
+callback_authentication_partial = partial(requires_authentication, module='callbacks')
+callbacks.before_request(callback_authentication_partial)
+
+
+from .views import notify  # noqa
```

Comment:
Can we add `app/callbacks/__init__.py : E402` to `.flake8` instead of `#noqa` for the full line or is the `#noqa` because it triggers both an `E402` and an `E401`?

`E402 module level import not at top of file`
`F401 'notify' imported but unused`

Reply:
Also we could glob match `__init__.py` files for F401 🤷‍♂️